### DISCUSSION TITLE
Spellbook Re-ordering

### DISCUSSION THREAD

**Discussion by feelwood** [runelite/runelite#16682] [Ideas]
Could we now get spellbook reordering with the new 3rd party plugin policy change?

**Comment by Nightfirecat** [comment]
What policy change has been made regarding spellbook reordering?

**Reply by feelwood** [reply]
Does this not extend to spellbooks as well?
![IMG_5227](https://github.com/runelite/runelite/assets/48904896/0e8c0be9-0abd-4597-a0c0-73dc908f48bb)
![IMG_5226](https://github.com/runelite/runelite/assets/48904896/1580a6ad-8b4c-4fd9-aae7-e190fe031067)

**Reply by Nightfirecat** [reply]
They have not lifted the restriction on spellbook reordering, regardless of whether we could achieve it at a technical level.

**Comment by Alexsuperfly** [comment]
It's still disallowed on https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1
